File: p1/c++/p1.cpp
/* IMPORT LIBRARIES */
#include <iostream>

/* INITIALIZE VARIABLES */
int num1 = 3;		// smaller num
int num2 = 5;		// larger num
int num_final = 10;	// limit num

/* MAIN PROGRAM */
int main()
{
	/* Assuming num_final > num1 */
	int sum = num1;
	int i;
	for(i = 4; i<num_final; i++)
	{
		if((i % num1 == 0) || (i % num2 == 0))
		{
			sum += i;
		}
	}
	printf("The summation of the integers up to %d are %d\n", num_final, sum);
	return 0;
}

File: p1/python/p1.py
# INITIALIZATIONS
num1 = 3        # small num
num2 = 5        # larger num
num_end = 1000    # limit num

# MAIN PROGRAM
if __name__ == "__main__":
    # Assuming num_end > num1
    summation = 3
    for i in range(4, num_end):
        if ((i % num1 == 0) or (i % num2 == 0)):
            summation += i
    
    print("Sum of the multiplies up to", num_end, "is", summation)

